srcpkgs/musikcube/template

```diff
@@ -1,6 +1,6 @@
 # Template file for 'musikcube'
 pkgname=musikcube
-version=0.99.4
+version=3.0.0
```

Comment:
3.0.1 available

Reply:
I see that it's indeed been committed to the musikcube repository a week ago, but it didn't exactly come out as a release yet